File: src/Kamil/Fsd.Kamil.InTheClassroom/Weekend2/Fsd.Cs.ConsoleApp/Models/Book.cs
namespace Fsd.Cs.ConsoleApp.Models
{
    public class Book
    {
        public string Title { get; set; }

        public int PageCount { get; set; }

        public BookType Type { get; set; }
        
        public Book(string title, int pageCount, BookType type)
        {
            Title = title;
            PageCount = pageCount;
            Type = type;
        }
    }
}

File: src/Kamil/Fsd.Kamil.InTheClassroom/Weekend6/data-generator/scripts/book.js
var app = app || {};

(function () {
  function Book(id, author, title, pageCount) {
    this.id = id;
    this.author = author;
    this.title = title;
    this.pageCount = pageCount;
  }

  app.types = {
    Book: Book
  };
})();
